### PR TITLE
Add info how to find deploy keys in dbt cloud

### LINKED ISSUE #3124: Clarify details on how to find deploy key in dbt cloud
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

Per [customer feedback](https://dbt-labs.slack.com/archives/C019QLR6EJE/p1680267653135539), this issue adds a line to clarify where a user can find their deploy key in dbt cloud.

zendesk ticket: https://dbtcloud.zendesk.com/agent/tickets/45322

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/docs/cloud/git/import-a-project-by-git-url#managing-deploy-keys

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by mirnawong1**
Per [customer feedback](https://dbt-labs.slack.com/archives/C019QLR6EJE/p1680267653135539), this issue adds a line to clarify where a user can find their deploy key in dbt cloud.

Closes #3124

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/cloud/git/import-a-project-by-git-url.md [hunk 1]
```diff
@@ -10,9 +10,17 @@ You must use the `git@...` or `ssh:..`. version of your git URL, not the `https:
 
 
 ## Managing Deploy Keys
-After importing a project by Git URL, dbt Cloud will generate a Deploy Key for your repository. You must provide this Deploy Key in the Repository configuration of your Git host. This Deploy Key should be configured to allow *read and write access* to the specified repositories.
 
-**Note**: Each dbt Cloud project will generate a different deploy key when connected to a repo, even if two projects are connected to the same repo. Both deploy keys will need to be supplied to your git provider.
+After importing a project by Git URL, dbt Cloud will generate a Deploy Key for your repository. To find the deploy key in dbt Cloud:
+
+1. Click the cog icon in the upper right-hand corner.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/git/import-a-project-by-git-url.md]: I think we are using 'gear' but forgive me if I'm mistaken
```suggestion
1. Click the gear icon in the upper right-hand corner.
```

File: website/docs/docs/cloud/git/import-a-project-by-git-url.md [hunk 2]
```diff
@@ -10,9 +10,17 @@ You must use the `git@...` or `ssh:..`. version of your git URL, not the `https:
...
+2. Click **Account Settings** --> **Projects** and select a project.
+3. Click the **Repository** link to the repository details page.
+4. Then copy the key under the **Deploy Key** section.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/git/import-a-project-by-git-url.md]:
```suggestion
4. Copy the key under the **Deploy Key** section. 
```

File: website/docs/docs/cloud/git/import-a-project-by-git-url.md [hunk 3]
```diff
@@ -10,9 +10,17 @@ You must use the `git@...` or `ssh:..`. version of your git URL, not the `https:
...
+After importing a project by Git URL, dbt Cloud will generate a Deploy Key for your repository. To find the deploy key in dbt Cloud:
+
+1. Click the cog icon in the upper right-hand corner.
+2. Click **Account Settings** --> **Projects** and select a project.
+3. Click the **Repository** link to the repository details page.
+4. Then copy the key under the **Deploy Key** section. 
+
+You must provide this Deploy Key in the Repository configuration of your Git host. Configure this Deploy Key to allow *read and write access* to the specified repositories.
+
+**Note**: Each dbt Cloud project will generate a different deploy key when connected to a repo, even if two projects are connected to the same repo. You will need to supply both deploy keys to your git provider.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/git/import-a-project-by-git-url.md]:
```suggestion
**Note**: Each dbt Cloud project will generate a different deploy key when connected to a repo, even if two projects are connected to the same repo. You will need to supply both deploy keys to your Git provider.
```

File: website/docs/docs/cloud/git/import-a-project-by-git-url.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/docs/cloud/git/import-a-project-by-git-url.md]: awesome, thank you Matt! i knew it wasn't cog

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**